### PR TITLE
SCSS code review

### PR DESCRIPTION
**Pull request by ckoga**
Code review for file change CSS -> SCSS
#### This code review is in order to look at issues [ 32, 9, 31 ]

- [ ] @spaceplesiosaur 
- [ ] @MattTuring

### CODE REVIEW

File: src/css/index.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*,
+*:before,
```
Comment (spaceplesiosaur) [inline src/css/index.scss]: I've never seen these *before and *after things in CSS before - can you add a screenshot of what this looks like?

File: src/css/styles.scss [hunk 2]
```diff
@@ -0,0 +1,180 @@
+*,
+*:before,
```
Comment (spaceplesiosaur) [inline src/css/styles.scss]: This is hard to do without a visual - maybe we can look at what it looks like in the morning before we officially merge!

File: src/css/styles.scss [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  }
+
+  .username {
+    font-weight: 600;
+  }
+}
+
+
+.grid {
+  columns: 3;
```
Comment (spaceplesiosaur) [inline src/css/styles.scss]: @MattTuring  noticed that another easy way to do this is with Flex.

File: src/scripts.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ import hydrationData from "../data/hydration";
 
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/normalize.css';
-import './css/styles.css';
+import './css/styles.scss';
```
Comment (spaceplesiosaur) [inline src/scripts.js]: They also wanted an .scss variable file, should we make one of those as well?  Or does styles.scss import the variables?

File: src/index.html [hunk 5]
```diff
@@ -3,8 +3,8 @@
```
Comment (spaceplesiosaur) [inline src/index.html]: This is all stuff added by the plugin in the config file, ignoring this.

File: src/css/index.scss (hunk 1 above)
Comment (ckoga) [reply src/css/index.scss]: ```:before , :after``` are <a href="https://css-tricks.com/almanac/selectors/a/after-and-before/">pseudo element selectors</a>  what I am saying here is that for all elements on the page I want to use box-sizing border box and for all child elements I want them to inherit this rule.  
here is the layout as is.  I do want to change the class of grid to use grid-template-column.
screencapture-localhost-8081-2019-10-21-09_04_59

**Comment by MattTuring** [comment]
This can be merged after the dist files are removed

**Pull request merged**